x-pack/plugins/uptime/public/components/monitor/status_details/availability_reporting/availability_reporting.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiBasicTable, EuiSpacer } from '@elastic/eui';
+import { Pagination } from '@elastic/eui/src/components/basic_table/pagination_bar';
+import { StatusTag } from './location_status_tags';
+import { TagLabel } from './tag_label';
+import { AvailabilityLabel, LastCheckLabel, LocationLabel } from '../translations';
+
+interface Props {
+  allLocations: StatusTag[];
+}
+
+export const AvailabilityReporting: React.FC<Props> = ({ allLocations }) => {
+  const [pageIndex, setPageIndex] = useState(0);
+
+  const cols = [
+    {
+      field: 'label',
+      name: LocationLabel,
+      truncateText: true,
+      render: (val: string, item: StatusTag) => {
+        return <TagLabel color={item.color} label={item.label} />;
+      },
+    },
+    {
+      field: 'availability',
+      name: AvailabilityLabel,
+      align: 'right' as const,
+      render: (val: number) => {
+        return <span>{val.toFixed(2)} %</span>;
```

Comment:
I checked with the localization team and they'd like to see strings like this localized also.

```suggestion
        return (
          <span>
            <FormattedMessage
              id="xpack.uptime.availabilityLabelText"
              defaultMessage="{value} %"
              values={{ value: val.toFixed(2) }}
              description="A percentage value, like 23.5%"
            />
          </span>
        );
```